raja-perf/README

```diff
@@ -0,0 +1,7 @@
+cd rambler
```

Comment:
is this supposed to be "rambler"?

Reply:
This is an old README file. It will be removed once all the raja-perf configs are moved to the new format